helm-chart/dash0-operator/values.yaml

```diff
@@ -552,3 +582,49 @@ operator:
     #     operator: "Exists"
     #     effect: "NoSchedule"
     deploymentTolerations: []
+
+    # The nodeAffinity for the collector Deployment pod (cluster-metrics-collector) managed by the operator. This can
+    # be used to constrain which nodes the collector pod will be scheduled on.
+    #
+    # See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
+    deploymentNodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: "dash0.com/enable"
+                operator: "NotIn"
+                values: ["false"]
+              - key: "kubernetes.io/os"
+                operator: "In"
+                values: ["linux"]
+
+  # Settings related to the target-allocator managed by the operator.
+  targetAllocator:
+    # Resource settings for the target-allocator container of the target-allocator deployment managed by the operator.
+    containerResources:
+      limits:
+        cpu: 200m
+        memory: 500Mi
+      requests:
+        cpu: 200m
+        memory: 128Mi
+
+    # An array of tolerations for the target-allocator deployment pods managed by the operator.
+    # This can be used to make sure that target-allocator pod can be scheduled on nodes where it would not be scheduled
+    # otherwise due to Kubernetes taints.
+    # Example:
+    # deploymentTolerations:
+    #   - key: "key1"
+    #     operator: "Equal"
+    #     value: "value1"
+    #     effect: "NoSchedule"
+    #   - key: "key2"
+    #     operator: "Exists"
+    #     effect: "NoSchedule"
+    tolerations: []
+
+    # The nodeAffinity for the target-allocator deployment managed by the operator. This can be used to constrain which
+    # nodes the target-allocator pod will be scheduled on.
+    #
+    # See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
+    nodeAffinity: {}
```

Comment:
q: How about setting the same default node affinities as for the other components? I think this would be good just for consistency, and I think both affinities would also make sense for the ta (restricting it to Linux nodes, and enabling users to use the `dash0.com/enable=false` anti-affinity. WDYT?